gobblin-core/src/main/resources/mysql/mysql.properties

```diff
@@ -16,7 +16,7 @@
 #
 
 # Source properties - source class to extract data from Mysql Source
-source.class=gobblin.source.extractor.extract.jdbc.MysqlSource
+source.class=gobblin.source.jdbc.MysqlSource
```

Comment:
Yes, but it's significantly harder to change both at the same time. For example, there tend to be a lot of jobs that use the MySqlSource, while adding the dependency requires changing a single file.

It is true that is a bad name, but we have to live with those decisions...